### ISSUE TITLE
Helm chart: Support for mixed clusters (Windows + Linux) 

### ISSUE THREAD

**Issue by SSvilen** [NetApp/trident#542]
The helm chart should have the option for setting tollerations.
Those would be important for mixed clusters - for instance with Rancher, where all pods by default are scheduled on the windows nodes. 
I could provide more information, if such is required. 
I could also implement those changes and open a  PR - just let me know.

**Comment by gnarl** [comment]
Hi @SSvilen,

Today we already enforce the node selection to only use amd64/Linux worker nodes. Every form of Trident install will achieve this default.

**Comment by SSvilen** [comment]
Hi @gnarl ,

unfortunately with mixes clusters in Rancher additional tolerations are required, in order to get a linux workload on linux nodes.
We ended up deploying Trident manually and editing every yaml file with the tolerations required.
I do not have experience with any other mixes clusters, but I believe the ability to set tolerations would be a nice feature.

**Comment by clintonk** [comment]
@SSvilen Would you mind sharing the additional tolerations you found were needed?

**Comment by SSvilen** [comment]
Hi @clintonk ,

```yaml
- effect: NoSchedule
   key: cattle.io/os
   operator: Equal
   value: linux
```

**Comment by gnarl** [comment]
This issue is resolved with commits [e167c7](https://github.com/NetApp/trident/commit/e167c7e92e143e4bb4a8373ff0dc9f41c374cd42) and [c7c167d](https://github.com/NetApp/trident/commit/c7c167da22657559ae2c330e3aa8ef7150300214) . This change will be included with the Trident 22.01 release.
